File: packages/sdk/sdk-py/examples/chat/image.py
import os
from phaseo import Phaseo

api_key = os.getenv("PHASEO_API_KEY")
if not api_key:
	raise RuntimeError("Set PHASEO_API_KEY")

client = Phaseo(api_key=api_key)

response = client.chat.completions.create({
	"model": "openai/gpt-5-nano",
	"messages": [
		{
			"role": "user",
			"content": [
				{"type": "text", "text": "What is in this image?"},
				{
					"type": "image_url",
					"image_url": {
						"url": "https://raw.githubusercontent.com/github/explore/main/topics/python/python.png"
					},
				},
			],
		},
	],
})

print(response)

File: packages/sdk/sdk-ts/examples/chat/image.ts
import { Phaseo } from "../../src";

const apiKey = process.env.PHASEO_API_KEY;
if (!apiKey) throw new Error("Set PHASEO_API_KEY");

async function main() {
	const client = new Phaseo({ apiKey });
	const response = await client.chat.completions.create({
		model: "openai/gpt-5-nano",
		messages: [
			{
				role: "user",
				content: [
					{ type: "text", text: "What is in this image?" },
					{
						type: "image_url",
						image_url: {
							url: "https://raw.githubusercontent.com/github/explore/main/topics/python/python.png",
						},
					},
				],
			},
		],
	});

	console.log(JSON.stringify(response, null, 2));
}

main().catch((err) => {
	console.error(err);
	process.exitCode = 1;
});
